moveit_ros/planning/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -164,7 +162,7 @@ bool MoveItCpp::loadPlanningPipelines(const PlanningPipelineOptions& options)
 
 moveit::core::RobotModelConstPtr MoveItCpp::getRobotModel() const
 {
-  return robot_model_;
+  return planning_scene_monitor_->getRobotModel();
```

Comment:
With regards to a recent review of yours this should be a header-only implementation. No strong feelings though.